src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -27,22 +29,51 @@ namespace Tizen.Security
     public static class PrivacyPrivilegeManager
     {
         private const string LogTag = "Tizen.Privilege";
-        private static Interop.PrivacyPrivilegeManager.RequestResponseCallback s_requestResponseCb;
+        private static Interop.PrivacyPrivilegeManager.RequestResponseCallback s_requestResponseCb =
+                       (Interop.PrivacyPrivilegeManager.CallCause cause, Interop.PrivacyPrivilegeManager.RequestResult result,
+                           string privilege, IntPtr userData) =>
+                        {
+                            try
+                            {
+                                if (s_responseMap.TryGetValue(privilege, out ResponseContext ctx) && ctx != null)
+                                {
+                                    ctx.FireEvent((CallCause)cause, (RequestResult)result);
+
+                                }
+                                else
+                                {
+                                    s_responseMap.Remove(privilege);
```

Comment:
You does not need call Remove, 
becuase, TryGetValue was truning false, it means privilege was not contained into dictionary